plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/handlers/RunAnalysisHandler.java

```diff
@@ -14,13 +26,49 @@
 	public Object execute(ExecutionEvent event) throws ExecutionException {
 		final AnalysisKickOff akf = new AnalysisKickOff();
 		IProject ip = Utils.getCurrentlySelectedIProject();
+		
+//		check if the open editor is in the same project
+		if (ip == Utils.getCurrentProject()) {

```

Comment:
This check fails if they are not the exact same object. I don't know if they always are, but I would prefer to not rely on this assumption.